### PR TITLE
SuperOTC format for font-source-han-sans

### PR DESCRIPTION
**Pull request by sttan**
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free. **[FIXED]**
- [x] `brew cask style --fix {{cask_file}}` left no offenses. 

------
See #519.

### CODE REVIEW

File: Casks/font-source-han-sans.rb [hunk 1]
```diff
@@ -1,37 +1,10 @@
-cask 'font-source-han-sans' do
+Cask 'font-source-han-sans' do
```
Comment (vitorgalvao) [inline Casks/font-source-han-sans.rb]: `Cask` needs to remain lowercase.

**Comment by vitorgalvao** [comment]
Please add the `name` stanza.

**Comment by sttan** [comment]
Done.

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**